Competitive_Programming/C++/Searching & Sorting/Heap_sort.md

```diff
@@ -0,0 +1,197 @@
+# Heap Application in Competitive Programming
```

Comment:
more over, Instead of naming it as HeapSort, findingKthLargestElement.md would makes more sense.
and place it under Heap Folder